apps/store/modules/asset-api.js

```diff
@@ -540,4 +540,24 @@ var responseProcessor = require('utils').response;
         paging.url = url;
         return paging;
     };
+    
+    api.formatSearchQuery = function (queryString) {
+        var searchQuery = "";
+        var qjson = parse('{' + queryString + '}');
+
+        var searchKeys = Object.keys(qjson);
+        if ((searchKeys.length === 1) && (searchKeys.indexOf("name") >= 0)) {
+            searchQuery += qjson[searchKeys.pop()];
+        }
+        else {
+            for (var keyIndex in searchKeys) {
+                var key = searchKeys[keyIndex];
+                var value = qjson[key];
+                searchQuery += key + ":" + value + " ";
+            }
+        }
+        searchQuery = searchQuery.trim();
+        return searchQuery;
+    };
+
```

Comment:
Thanks @thushara35 , I have changed it in [this commit](https://github.com/wso2/carbon-store/pull/844/commits/993a3cd2fad8bce67dd7caca21bcc51675b3d9e5)